judge/models/contest.py

```diff
@@ -179,6 +186,8 @@ def can_see_own_scoreboard(self, user):
             return False
         if not self.show_scoreboard and not self.is_in_contest(user):
             return False
+        if self.scoreboard_visibility == self.SCOREBOARD_AFTER_PARTICIPATION and not self.is_in_contest(user):
```

Comment:
Do we need this check? Shouldn't it be covered by the check 2 lines above?

Reply:
I just checked, yes that statement seems useless